src/org/zaproxy/zap/extension/ascanrulesAlpha/resources/help/contents/ascanalpha.html

```diff
@@ -16,6 +16,11 @@
 <H2>Cookie Slack Detector</H2>
 Tests cookies to detect if some have no effect on response size when omitted, especially cookies containing the name "session" or "userid"
 
+<H2>ELMAH Scanner</H2>
+Tests to see if the  Error Logging Modules and Handlers (elmah.axd) HTTP Module is available. Although this module is handy for
```

Comment:
- There's an extra space that could be removed.
 - Worth mentioning the technologies targeted by the scanner.